src/libraries/System.IO.Ports/src/System.IO.Ports.csproj

```diff
@@ -120,8 +120,10 @@
   <ItemGroup Condition="'$(TargetsLinux)' == 'true' or '$(TargetsOSX)' == 'true' or '$(TargetsFreeBSD)' == 'true'">
     <Compile Include="System\IO\Ports\SafeSerialDeviceHandle.Unix.cs" />
     <Compile Include="System\IO\Ports\SerialStream.Unix.cs" />
-    <Compile Include="Interop\Unix\Interop.Termios.cs" />
-    <Compile Include="Interop\Unix\Interop.Serial.cs" />
+    <Compile Include="$(CommonPath)Interop\Unix\Interop.Termios.cs"
```

Comment:
These files should be in `System.IO.Ports.Native` subdirectory - Interop\Unix\System.IO.Ports.Native\Interop.Termios.cs.